src/.pre-commit-config.yaml.jinja

```
@@ -85,6 +85,8 @@ exclude: |
   readme/.*\.(rst|md)$|
   # Ignore build and dist directories in addons
   /build/|/dist/|
+  # Ignore test files in addons
+  /tests/.*\.(wamas|txt|xml)$|
```

```suggestion
  /tests/fixtures/*|
```
I would ignore everything in a given dir.
If by chance someone wants to have formatted xmls there the it will be their choices.
Also, I would not tie this to any specific extension.

---

hi @simahawk the PR has been updated as your request